splitcidr/main.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/fatih/color"
+	"github.com/ipinfo/cli/lib"
+	"github.com/ipinfo/cli/lib/complete/install"
+	"github.com/spf13/pflag"
+)
+
+var progBase = filepath.Base(os.Args[0])
+var version = "1.0.0"
+
+func printHelp() {
+	fmt.Printf(
+		`Usage: %s splitcidr <cidr> <split>
+
+Description:
+  splits a larger CIDR into smaller CIDRs.
+
+  $ %[1]s splitcidr 8.8.8.0/24 25
```

Comment:
In this case `%[1]s` is already `splitcidr`.

```suggestion
  $ %[1]s 8.8.8.0/24 25
```